app/models/broadcast.rb

```diff
@@ -3,6 +3,7 @@ class Broadcast < ActiveRecord::Base
 
   has_many(:broadcast_addressings, :inverse_of => :broadcast, :dependent => :destroy)
   has_many(:recipients, :through => :broadcast_addressings, :source => :user)
+  has_many(:sms_messages, :dependent => :destroy, :class_name => 'Sms::Message')
```

Comment:
I was trying to do `Broadcast.where(...).destroy_all` in the console and it was failing. I can drop it if you like.